Modules/Scripted/ScreenCapture/ScreenCapture.py

```diff
@@ -1050,10 +1083,20 @@ def captureImageFromView(self, view, filename, transparentBackground=False):
       imageClipper.Update()
       capturedImage = imageClipper.GetOutput()
 
-    writer = self.createImageWriter(filename)
-    writer.SetInputData(self.addWatermark(capturedImage))
-    writer.SetFileName(filename)
-    writer.Write()
+    capturedImage = self.addWatermark(capturedImage)
+    if isinstance(volumeNode, slicer.vtkMRMLVectorVolumeNode):
+      ijkToRas = vtk.vtkMatrix4x4()
+      ijkToRas.SetElement(0, 0, -1)
+      ijkToRas.SetElement(1, 1, -1)
+      volumeNode.SetIJKToRASMatrix(ijkToRas)
```

Comment:
@lassoan This still isn't changing the results of the output for some reason. I'm still getting that flipped image. Am I doing some wrong here?

Reply:
This should work well. It should make the image appear correctly in the red slice view if you use Slicer default view orientations. Do you see that ijkToRas matrix makes a difference?

Reply:
I have set the ijkToRas matrix and you can see below that it is returning the (-1,-1,1,1) diagonal, but the output still looks flipped. See image below
![image](https://user-images.githubusercontent.com/15837524/114767566-c93d5700-9d35-11eb-9604-51441bca71b4.png)

Reply:
Do you see that changing ijkToRas matrix makes a difference? (if you set diag(1,1,1,1) the image is flipped compared to diag (-1,1,1,1))

Can you take a screenshot with "Axis" orientation marker displayed?

Reply:
Ok I have identified what has made things confusing. Not sure what to do to make it both valid when viewing in slice views and upon save to PNG. 

With no SetIJKToRAS calls in the capture image method:
- The VectorVolume as viewed in the slice views needs a (-1,1,1,1) IJKToRAS matrix to flip it to the expected view
![image](https://user-images.githubusercontent.com/15837524/114773581-3a343d00-9d3d-11eb-9582-14ee8f094fea.png)
- However when saving the node to PNG using the Slicer save dialog, it actually needs a (1,-1,1,1) IJKToRAS matrix to flip it to the expected view.
![VectorVolume](https://user-images.githubusercontent.com/15837524/114773716-6485fa80-9d3d-11eb-8f40-97edc5824b35.png)